torch/_inductor/scheduler.py

```diff
@@ -602,6 +606,90 @@ def get_buf_bytes(buf: Optional[Union[ir.Buffer, ir.TensorBox]]) -> int:
 
         return node_bytes
 
+    def get_benchmark_runtime(self) -> float:
```

Comment:
Let's put this function into `utils.py`. I understand that `get_estimated_runtime` is here, but for this project, let's try to provide a minimal interface in `scheduler.py` and put most of our code in the dedicated folder `simple_fsdp`. If later on we need to upstream to pytorch main branch, we can revisit where's the best place to put them.